Source/FakeItEasy.Tests/Core/ArgumentValueFormatterTests.cs

```diff
@@ -139,7 +141,25 @@ public void Should_prefer_exact_type_formatter_to_interface_formatter()
             var result = this.formatter.GetArgumentValueAsString("string value");
 
             // Assert
-            Assert.That(result, Is.EqualTo("a string"));
+            result.Should().Be("a string");
+        }
+
+        [Test]
+        public void Provided_formatters_should_have_negative_priority()

```

Comment:
So we could.

1. update to FA 4.2.1, or
2. change the test so that it reports the failing types, perhaps by using a test case source, adding an always-good formatter (dummy factory, options builder) if no test targets are found
3. something that I can't think of right now

If you have a preferred approach, I'm happy to take it.